src/components/cardOfCourse/CardOfCourse.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Rating from './rating';
+import CoursePhoto from './coursePhoto';
+import CourseInfo from './courseInfo';
+import { Box } from '@mui/material';
+import GroupsIcon from '@mui/icons-material/Groups';
+import CoursePrice from './coursePrice';
+import IconButton from '@mui/material/IconButton';
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import { Link } from 'react-router-dom';
+import './card.scss';
+
+const CardOfCourse = () => {
+  const [isFavorite, setIsFavorite] = useState(false);
```

Comment:
```suggestion
  const [isFavorite, setIsFavorite] = useState<boolean>(false);
```
Please write type of data in useState)))

Reply:
Of course, I've already added)